### PR TITLE
20397 Added new alert

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#20397

*Description of changes:*
- app version = 7.1.5
- added Corporate Online alert
- added/updated unit tests
- deleted unneeded router from EntityHeader test suite
- deleted unneeded router from Entity Menu test suite

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/EntityInfo/EntityMenu.vue [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
+        transition="fade-transition"
+      >
+        <template #activator="{ on }">
+          <v-btn
+            id="colin-link-button"
+            small
+            text
+            color="primary"
+            :href="getCorporateOnlineUrl"
+            target="_blank"
```
Comment (severinbeauvais) [inline src/components/EntityInfo/EntityMenu.vue:26]: I checked with Yui and, yes, we want COLIN to open in a new tab/window.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-639-bizes7em.web.app

**Review by PaulGarewal — APPROVED** [review]
Looks great 👍 been learning a lot from your recent PRs 🤓

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-639-bizes7em.web.app

**Review by ketaki-deodhar — APPROVED** [review]
Looks good Sev! 👍

**Pull request merged**